Explorer/Assets/DCL/PluginSystem/Global/ExplorePanelPlugin.cs

```diff
@@ -0,0 +1,69 @@
+using Arch.SystemGroups;
+using Cysharp.Threading.Tasks;
+using DCL.AssetsProvision;
+using DCL.ExplorePanel;
+using DCL.Navmap;
+using MVC;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using static UnityEngine.Object;
+
+namespace DCL.PluginSystem.Global
+{
+    public class ExplorePanelPlugin : IDCLGlobalPlugin<ExplorePanelPlugin.ExplorePanelSettings>
+    {
+        private readonly IAssetsProvisioner assetsProvisioner;
+        private readonly IMVCManager mvcManager;
+
+        public ExplorePanelPlugin(IAssetsProvisioner assetsProvisioner, IMVCManager mvcManager)
+        {
+            this.assetsProvisioner = assetsProvisioner;
+            this.mvcManager = mvcManager;
+        }
+
+        public async UniTask InitializeAsync(ExplorePanelSettings settings, CancellationToken ct)
+        {
+            mvcManager.RegisterController(new ExplorePanelController(
+                ExplorePanelController.Preallocate(
+                    (await assetsProvisioner.ProvideMainAssetAsync(settings.explorePanelPrefab, ct: ct)).Value.GetComponent<ExplorePanelView>(), null, out var explorePanelView)));

```

Comment:
Also if you used `ComponentReference<TComponent>` you could call `ProvideMainAsset` without `GetComponent`